src/section/mod.rs

```diff
@@ -134,6 +151,14 @@ impl Section {
             .chain
             .push(new_elders_info.proof.public_key, new_key_proof.signature)
         {
+            error!(
+                "fork attempt detected: new key: {:?}, new prefix: ({:b}), expected current key: {:?}, current chain: {:?}, current prefix: ({:b})",
```

Comment:
alreay have key or signature check failure will also cause this error, why just say `fork attempt detected` ?

Reply:
Yeah, that's true. Not sure it's worth distinguishing these case though. See also the previous comment reply.